linux/alsa/alsa_driver.c

```diff
@@ -1055,6 +1055,12 @@ alsa_driver_get_channel_addresses (alsa_driver_t *driver,
 	return 0;
 }
 
+static int
+alsa_driver_stream_start(snd_pcm_t *pcm, bool is_capture)
```

Comment:
What purpose does it serve to add an unused function parameter? If the idea is to make function calls somehow more uniform, then I think enforcing such an ideal of uniformity even for internal static functions crosses the line from reasonable to dogmatic.

Is there something I'm not seeing here? Does adding `bool is_capture` actually lay groundwork for an actually useful change down the line? How?

Reply:
yeah this is a groundwork for additional features. iirc the general feature is adding multiple playback/capture devices that can be started in alsa_driver. this was done some years ago and I still have the patches laying around somewhere on my drive.

it didnt get merged in previous PR due to large amount of changes, so I split in and this is one of the PRs that is a pre-requisite for that work.